scripts/index.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs')
+const path = require('path')
+//CO2e -> kilowattHours * emissions[region];
+function updateCO2eData() {
```

Comment:
i would suggest making this async and just naming the function `main` to align with our other scripts

Reply:
with this change, I would also request changing the name of the file from index.js to something like 'updateMockData.js'